MapboxNavigationUI/NavigationUI.swift

```diff
@@ -0,0 +1,80 @@
+import UIKit
+import MapboxDirections
+import Mapbox
+
+let MBSecondsBeforeResetTrackingMode:TimeInterval = 25.0
+
+extension UIColor {
+    fileprivate class var defaultTint: UIColor { get { return #colorLiteral(red: 0.1411764771, green: 0.3960784376, blue: 0.5647059083, alpha: 1) } }
+    fileprivate class var defaultTintStroke: UIColor { get { return #colorLiteral(red: 0.05882352963, green: 0.180392161, blue: 0.2470588237, alpha: 1) } }
+    fileprivate class var defaultPrimaryText: UIColor { get { return #colorLiteral(red: 45.0/255.0, green: 45.0/255.0, blue: 45.0/255.0, alpha: 1) } }
+    fileprivate class var defaultSecondaryText: UIColor { get { return #colorLiteral(red: 0.4509803922, green: 0.4509803922, blue: 0.4509803922, alpha: 1) } }
+    fileprivate class var defaultLine: UIColor { get { return #colorLiteral(red: 151.0/255.0, green: 151.0/255.0, blue: 151.0/255.0, alpha: 0.6) } }
+}
+
+@objc(MBNavigationUI)
+public class NavigationUI: NSObject {
+    
+    public static let shared = NavigationUI()
+    
+    override public class func initialize() {
+        _ = UIStoryboard(name: "Navigation", bundle: Bundle.navigationUI)
```

Comment:
This was a workaround for "Bundle not loaded/Could not find storyboard" crash but we're not using a resource bundle so should be safe to remove.